rosbag2_cpp/include/rosbag2_cpp/writers/sequential_writer.hpp

```diff
@@ -113,6 +113,11 @@ class ROSBAG2_CPP_PUBLIC SequentialWriter
   // Used in bagfile splitting; specifies the best-effort maximum sub-section of a bagfile in bytes.
   uint64_t max_bagfile_size_;
 
+  // Intermediate cache to write multiple messages into the storage.
+  // `max_cache_size` is the amount of messages to hold in storage before writing to disk.
+  size_t max_cache_size_;
```

Comment:
`max_cache_size` is a `uint64_t` in `StorageOptions`

Reply:
good catch!